.github/workflows/official-pr.yml

```diff
@@ -36,7 +35,7 @@ jobs:
       - name: Generate Stackbrew for diff
         run: |
           cd docker-node
-          ./generate-stackbrew-library.sh > ../official-images/library/node
+          ./stackbrew.js > ../official-images/library/node
```

Comment:
Right, so since this is changing something triggered by the `pull_request_target`, its not getting triggered till the next time it's run. Need to send in a new minor PR to update the upstream one